Library/Formula/duck.rb

```diff
@@ -0,0 +1,25 @@
+require "formula"
+
+class Duck < Formula
+  homepage "https://duck.sh/"
+  # Reference to the CLI version of Cyberduck
+  url "https://update.cyberduck.io/cli//duck-4.6.2.16269.tar.gz"
```

Comment:
The URI is valid. The double slash comes form concatenating and an optional in between path component that we use for snapshot builds.